src/modules/rlm_python/rlm_python.c

```diff
@@ -817,7 +824,7 @@ static int mod_instantiate(CONF_SECTION *conf, void *instance)
 	 */
 	code = do_python(inst, NULL, inst->instantiate.function, "instantiate", false);
 	if (code >= 0) {
-		return code;
+		return 0;
```

Comment:
No point returning code here; it won't currently be used, so safer to just return 0.